launch/launch/actions/execute_process.py

```diff
@@ -37,13 +37,17 @@
 from .emit_event import EmitEvent
 from .opaque_function import OpaqueFunction
 from .timer_action import TimerAction
+
+from .. import logging
```

Comment:
This is a bit confusing. i kept thinking you were calling the python logging module below. I'd recommend either renaming this module (locally with `as ....` or at the module itself), or consider using non-relative import for this particular case, i.e. `launch.logging.*` everywhere.

Reply:
Non-relative import it is then.

Reply:
Sounds good.